dotnet/src/SemanticKernel.Abstractions/IKernel.cs

```diff
@@ -201,5 +191,20 @@ public interface IKernel
     [EditorBrowsable(EditorBrowsableState.Never)]
     ISKFunction Func(string pluginName, string functionName);
 
+    [Obsolete("Methods, properties and classes which include Skill in the name have been renamed. Use Kernel.ImportFunctions instead. This will be removed in a future release.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CS1591
+    IDictionary<string, ISKFunction> ImportSkill(object functionsInstance, string? pluginName = null);
+#pragma warning restore CS1591
+
+    /// <summary>
+    /// Set the semantic memory to use
+    /// </summary>
+    /// <param name="memory">Semantic memory instance</param>
+    /// <inheritdoc/>
+    [Obsolete("Memory functionality will be placed in separate Microsoft.SemanticKernel.Plugins.Memory package. This will be removed in a future release.")]
```

Comment:
Again we need to provide sufficient information so a developer knows how to change from the obsolete API.